.github/workflows/checks.yaml

```diff
@@ -465,3 +465,107 @@ jobs:
           "$(go env GOROOT)"/bin/go run github.com/google/go-licenses@5348b744d0983d85713295ea08a20cca1654a45e
           check --disallowed_types=forbidden --include_tests
           ./examples
+  migrate:
+    name: goose migrate up and down
+    runs-on: ubuntu-22.04 # Consistently Linux
+    services:
+      postgres:
+        image: postgres:15
+        env:
+          POSTGRES_USER: testuser
+          POSTGRES_PASSWORD: testpass
+          POSTGRES_DB: testdb
+        ports:
+          - "5432:5432"
+        options: >-
+          --health-cmd="pg_isready -U testuser -d testdb" --health-interval=10s --health-timeout=5s --health-retries=5
+    steps:
+      - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.4.2
+        with:
+          fetch-depth: 0
+          persist-credentials: false
+
+      - name: Install goose
+        run: go install github.com/pressly/goose/v3/cmd/goose@latest
```

Comment:
This version should be pinned to the version in service/go.mod file, not the latest, as any breaking change would break our workflow

Reply:
```suggestion
        run: go install github.com/pressly/goose/v3/cmd/goose@3.24.3
```